opentelemetry/src/common.rs

```diff
@@ -556,13 +529,13 @@ impl InstrumentationLibraryBuilder {
         self
     }
 
-    /// Create a new [InstrumentationLibrary] from this configuration
-    pub fn build(self) -> InstrumentationLibrary {
-        InstrumentationLibrary {
+    /// Create a new [InstrumentationScope] from this configuration
+    pub fn build(self) -> InstrumentationScope {
+        InstrumentationScope {
             name: self.name,
             version: self.version,
             schema_url: self.schema_url,
-            attributes: self.attributes.unwrap_or_default(),
+            attributes: Cow::Owned(self.attributes.unwrap_or_default()),
```

Comment:
Just to confirm, if users want to take advantage of borrowed attributes, they can bypass the builder pattern and directly initialize the `InstrumentationScope` struct.

Reply:
Reverted `attributes` to its initial type as per comments above.